### ISSUE TITLE
How do i use this, is it working?

### ISSUE THREAD

**Issue by Usulyre** [syniox/ddfi-vapoursynth#1]
Hi, is this working? Does it need some dependencies to work?

**Comment by syniox** [comment]
(sorry for my bad english.)
If more than 3 frames(inclusive) are the same, it's broken. Otherwise it might be working.
I'm sorry that I'm busy and can't fix it at the moment.

You need mvsfunc for the script, svp for ddfi_svp function, mvtools for ddfi_mv function.
So you can just install mvsfunc+svp or mvsfunc+mvtools since the two function do the same work.

**Comment by Usulyre** [comment]
Does this change the frame rate with frame interpolation?

**Comment by syniox** [comment]
> Does this change the frame rate with frame interpolation?

Nope, it just tries to replace the duplicated frames with interpolated frames

you can see demo here(almost the same function):
https://github.com/Mr-Z-2697/ddfi-rife
